README.rst

```diff
@@ -31,7 +31,7 @@ Initial Setup: steps and confirmation
 
 Note: this setup process is temporary; we will be working on a Tutor plugin
 
-.. code-block::
+.. code-block:: shell
```

Comment:
@pshiu: Curious if you get the same error with `none` or `text`?

Reply:
(Replied in https://github.com/edx/commerce-coordinator/pull/33#discussion_r900304662)